adobe/react-spectrum
# Fix Android TalkBack double tap behavior with TableView/ListView highlight selection

Closes #2512 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Go to the ListView/TableView "selectionStyle: highlight" story
2. Using Android TalkBack, select a row via moving the cursor to a row cell and double tapping
3. Move the cursor to a new row and double tap. Verify that the previously selected row isn't deselected and that the new row is selected as well
4. Go to the ListView/TableView "selectionStyle: highlight, onAction" story 
5. Move the cursor to the row. If in the ListView story do this by tapping on a row and swiping left once (this moves the cursor from the cell to the row) 
6. Double tap. Verify that onAction is fired and the row isn't selected

## 🧢 Your Project:

RSP
